_boards_cm/xtrx-pcie-sdr.md

```diff
@@ -0,0 +1,11 @@
+---
+layout: board
```

Comment:
This one is actually a card - can you put it in with the cards and update the layout to match?